net-misc/networkmanager/metadata.xml

```diff
@@ -10,6 +10,7 @@
     <flag name="connection-sharing">Support connection sharing (uses <pkg>net-dns/dnsmasq</pkg>)</flag>
     <flag name="dhclient">Use dhclient from <pkg>net-misc/dhcp</pkg> for getting an IP via DHCP</flag>
     <flag name="dhcpcd">Use <pkg>net-misc/dhcpcd</pkg> for getting an IP</flag>
+    <flag name="gnutls">Use <pkg>net-libs/gnutls</pkg> instead of <pkg>dev-libs/nss</pkg> for cryptography</flag>
```

Comment:
No, this needs really careful consideration and analysis of what the upstream preferred library is and to go from there. It should not be done simply to remove REQUIRED_USE and it requires justification.